chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomsReceivedIntegrationTest.kt

```diff
@@ -121,6 +125,7 @@ internal object JoinedRoomsReceivedIntegrationTest : Spek({
 
             it("then ReconnectJoinedRoom is dispatched") {
                 verify(exactly = 1) {
+                    dispatcher(CurrentUserReceived(userApiTypeMapper.toUserInternalType(simpleUser)))
```

Comment:
this test should not be testing for that, let's remove it and `confirmVerified` below, applies to more places

Reply:
Hmm. I'm not sure what the Integration test *should* be doing currently. Have asked on Slack and will follow up here with what we decide.